src/core_plugins/kibana/public/visualize/listing/create_or_delete_button.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+import { CreateButtonLink } from 'ui_framework/components/button/create_button_link';
+import { DeleteButton } from 'ui_framework/components/button/delete_button';
+
+export function CreateOrDeleteButton({ showCreate, doDelete }) {
```

Comment:
Can we avoid destructuring `props`? I've often had to define a function inside of a stateless functional component, and it's easier to differentiate between these kinds of functions and those provided by props when they are referred to as `functionName` and `props.functionName`.